winui/AvatarView/content-types.md

```diff
@@ -7,4 +7,305 @@ control: AvatarView
 documentation: ug
 ---
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+## Content Types
+
+The `AvatarView` control allows you to display the view in five different ways:
+
+* `Default` - Displays the `Avatar1` character image when initializing without any other source such as image, initials or group.
```

Comment:
@MadhumithanJayapathi The AvatarView control supports five different types of content: Default, Initials, CustomImage, AvatarCharacter, and GroupView.

Reply:
@Manivannan-E Changed the content as mentioned.